tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd_ucc_async.py

```diff
@@ -0,0 +1,37 @@
+import os
+import logging
+
+import torch
+from torch.testing._internal import common_utils
+from torch.testing._internal import common_distributed
+
+from apex.transformer.testing.distributed_test_base import UccDistributedTestBase
+
+from test_pipeline_parallel_fwd_bwd import PipelineParallelForwardBackwardTestBase
+
+logging.getLogger("torch").setLevel(logging.WARNING)
+logging.getLogger("apex").setLevel(logging.WARNING)
+
+weight_coeff = 1024
+
+class UccPipelineParallelForwardBackwardUnderLoadTest(UccDistributedTestBase, PipelineParallelForwardBackwardTestBase):
```

Comment:
What does class test?

Reply:
There are two main reasons for existence of this class. Compared to the original `test_pipeline_parallel_fwd_bwd.py` the load here is much higher, so one would be able to see the operations in profiler (nsight systems). Second, when running `test_pipeline_parallel_fwd_bwd.py -k test_learning_async_pipelining_without_interleaving` it will run the tests for NCCL and NCCL/UCC as well, which is often unwanted.

Reply:
> Second, when running `test_pipeline_parallel_fwd_bwd.py -k test_learning_async_pipelining_without_interleaving` it will run the tests for NCCL and NCCL/UCC as well, which is often unwanted.

It's because the expression is a bit too ambiguous. For example, `UccPipelineParallelForwardBackwardTest.test_learning_async_pipelining_without_interleaving` wold only run one test for unittest. For pytest "test_learning_async_pipelinign_without_interleaving and Ucc" would only run one case.

Reply:
Do we expect this test to uncover any additional failures that would not be covered by the previous use-cases, or would it be for profiling purposes only? (Would it fit under the umbrella of a slow test that is usually skipped?)

Reply:
The main purpose of this file was to do the profiling. And do I think this is not the best folder for hosting it. I am thinking of moving it somewhere to a new location like `apex/profiling/transformer` or something similar. 
@crcrpar @eqy what do you think?

Reply:
I'd put it in tests/L1/transformer and I'm not sure if it's needed to make this file testable.